.github/dependabot.yml

```diff
@@ -5,45 +5,72 @@
 
 version: 2
 updates:
+  # JAVASCRIPT package.json updates
 
   - package-ecosystem: "npm"
-    # Files stored in `frontend` workspace
     directory: "/frontend"
     schedule:
       interval: "weekly"
     open-pull-requests-limit: 15
 
   - package-ecosystem: "npm"
-    # Files stored in `auth` directory
     directory: "/auth"
     schedule:
       interval: "weekly"
     open-pull-requests-limit: 15
 
+  # PHP composer.json package updates
+
   - package-ecosystem: "composer"
-    # Files stored in `api` directory
     directory: "/api"
     schedule:
       interval: "weekly"
     open-pull-requests-limit: 15
 
   - package-ecosystem: "composer"
-    # Files stored in `admin` directory
     directory: "/frontend/admin"
     schedule:
       interval: "weekly"
     open-pull-requests-limit: 15
 
   - package-ecosystem: "composer"
-    # Files stored in `auth` directory
     directory: "/auth"
     schedule:
       interval: "weekly"
     open-pull-requests-limit: 15
 
   - package-ecosystem: "composer"
-    # Files stored in `talentsearch` directory
     directory: "/frontend/talentsearch"
     schedule:
       interval: "weekly"
     open-pull-requests-limit: 15
+
+  # DOCKER image updates
+
+  - directory: "/api"
+    package-ecosystem: "docker"
+    schedule:
+      interval: "weekly"
+    open-pull-requests-limit: 0
```

Comment:
unlike npm, updates so rare that left unlimited

Reply:
Makes sense.